api/config/service/auditsink/slack/v1/slack.proto

```diff
@@ -13,4 +13,17 @@ message SlackConfig {
   string channel = 2 [ (validate.rules).string = {min_bytes : 1} ];
 
   clutch.config.service.audit.v1.Filter filter = 3;
+  Override override = 4;
```

Comment:
should this be flattened to `repeated CustomSlackMessage overrides`? do you anticipate adding anything else to Override?

Reply:
flattened
don't plan on adding anything else to Override